src/frontend/src/test-e2e/pinAuth.test.ts

```diff
@@ -61,3 +110,63 @@ test("Should not prompt for PIN after deleting temp key", async () => {
     await FLOWS.login(userNumber, DEVICE_NAME1, browser);
   }, APPLE_USER_AGENT);
 }, 300_000);
+
+test("Log into client application, after PIN registration", async () => {
+  await runInBrowser(async (browser: WebdriverIO.Browser) => {
+    const pin = "123456";
+
+    const demoAppView = new DemoAppView(browser);
+    await demoAppView.open(TEST_APP_NICE_URL, II_URL);
+    await demoAppView.waitForDisplay();
+    expect(await demoAppView.getPrincipal()).toBe("2vxsx-fae");
+    await demoAppView.signin();
+    await switchToPopup(browser);
+    await FLOWS.registerPinNewIdentityAuthenticateView(pin, browser);
+    await waitToClose(browser);
+    await demoAppView.waitForDisplay();
+    const principal = await demoAppView.getPrincipal();
+    expect(principal).not.toBe("2vxsx-fae");
```

Comment:
Where does `"2vxsx-fae"` come from?

Reply:
It's the anonymous principal. I replaced it with a call to `Principal.anonymous()`.